src/RimDev.Stuntman.AspNetCore/common/StuntmanOptions.cs

```diff
@@ -119,11 +119,11 @@ public StuntmanOptions AddUsersFromJson(string pathOrUrl)
                 ? _stuntmanOptionsRetriever.GetStringFromLocalFile(uri)
                 : _stuntmanOptionsRetriever.GetStringUsingWebClient(uri);
 
-            var users = JsonConvert.DeserializeObject<IEnumerable<StuntmanUser>>(
+            var users = JsonConvert.DeserializeObject<StuntmanServerResponse>(
```

Comment:
This change is due to how the server-endpoint actually returns the data. Ref. https://github.com/ritterim/stuntman/blob/5f74e22091c82c57c3124a715cccb4342477e3e7/src/Core/IAppBuilderExtensions.cs#L107-L114